packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+    Tooltip,
+} from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import { IconChevronDown } from '@tabler/icons-react';
+import React, { FC, useMemo } from 'react';
+import { useTableStyles } from '../../hooks/styles/useTableStyles';
+import MantineIcon from '../common/MantineIcon';
+import {
+    Column,
+    formatTime,
+    getLogStatusIcon,
+    getSchedulerIcon,
+    getSchedulerLink,
+    Log,
+    SchedulerItem,
+} from './SchedulersView';
+
+type LogsProps = {
+    projectUuid: string;
+    schedulers: SchedulerWithLogs['schedulers'];
+    logs: SchedulerWithLogs['logs'];
+    users: SchedulerWithLogs['users'];
+    charts: SchedulerWithLogs['charts'];
+    dashboards: SchedulerWithLogs['dashboards'];
+};
+
+const Logs: FC<LogsProps> = ({
+    projectUuid,
+    schedulers,
+    logs,
+    users,
+    charts,
+    dashboards,
+}) => {
+    const { classes, theme } = useTableStyles();
+    const [opened, { toggle }] = useDisclosure(false);
+
+    const columns = useMemo<Column[]>(() => {
+        const currentLogs = (item: SchedulerItem, targets: Log[]) => {
+            return targets.filter(
+                (target) => target.schedulerUuid === item.schedulerUuid,
+            );
+        };
+        const handleScheduledDeliveryLogs = (
+            item: SchedulerItem,
+            targets: Log[],
+        ) =>
+            currentLogs(item, targets).filter(
+                (log) => log.task === 'handleScheduledDelivery',
+            );
+
+        const sendNotificationLogs = (item: SchedulerItem, targets: Log[]) =>
+            currentLogs(item, targets).filter(
+                (log) =>
+                    log.task === 'sendEmailNotification' ||
+                    log.task === 'sendSlackNotification',
+            );
+        return [
+            {
+                id: 'name',
+                label: 'Name',
+                cell: (item) => {
+                    const user = users.find(
+                        (u) => u.userUuid === item.createdBy,
+                    );
+                    const chartOrDashboard = item.savedChartUuid
+                        ? charts.find(
+                              (chart) =>
+                                  chart.savedChartUuid === item.savedChartUuid,
+                          )
+                        : dashboards.find(
+                              (dashboard) =>
+                                  dashboard.dashboardUuid ===
+                                  item.dashboardUuid,
+                          );
+                    return (
+                        <Anchor
+                            sx={{
+                                color: 'unset',
+                                ':hover': {
+                                    color: 'unset',
+                                    textDecoration: 'none',
+                                },
+                            }}
+                            href={getSchedulerLink(item, projectUuid)}
```

Comment:
if the link is internal we should try using react router navigation instead of using href.